src/briefcase/platforms/iOS/xcode.py

```diff
@@ -362,7 +371,9 @@ def run_app(self, app: BaseConfig, udid=None, **kwargs):
                 "--style",
                 "compact",
                 "--predicate",
-                f'senderImagePath ENDSWITH "/{app.formal_name}"',
+                f'senderImagePath ENDSWITH "/{app.formal_name}"'
+                f' OR (processImagePath ENDSWITH "/{app.formal_name}"'
+                ' AND senderImagePath ENDSWITH "-iphonesimulator.so")',
```

Comment:
Why not just include all messages where `processImagePath ENDSWITH "/{app.formal_name}"`? Is it because it would be too noisy? If so, it would be worth explaining that in a comment.

Reply:
The *process* will generate lots of messages that aren't user-originated (all the usual noise generated by the device notifying system-level events). We only care about the messages that come from the user's code (the sender). 

We could *almost* get away with *just* filtering on the `senderImagePath`. This would work on the simulator, and on any phone that only had 1 Python app running; but if you had a second app generating log messages, those would end up being interspersed with with the app you were trying to log.

Regardless - I'll add a note about this.